config/config.go

```diff
@@ -15,8 +15,9 @@ type OAuthConfig struct {
 
 // DisplayConfig defines the display switches shape
 type DisplayConfig struct {
-	Verbose bool `json:"verbose"`
-	QR      bool `json:"qr"`
+	DisableDirectoryListing bool `json:"disabledirectorylisting"`
```

Comment:
please use camelCase syntax
```suggestion
	DisableDirectoryListing bool `json:"disableDirectoryListing"`
```